skimage/transform/__init__.pyi

```diff
@@ -40,44 +40,24 @@ __all__ = [
     'pyramid_expand',
     'pyramid_gaussian',
     'pyramid_laplacian',
+    'tps_transform',
+    'tps_warp'
 ]
 
-from .hough_transform import (
-    hough_line,
-    hough_line_peaks,
-    probabilistic_hough_line,
-    hough_circle,
-    hough_circle_peaks,
-    hough_ellipse
-)
-from .radon_transform import (
-    radon, iradon, iradon_sart,
-    order_angles_golden_ratio
-)
+from ._geometric import (AffineTransform, EssentialMatrixTransform,
+                         EuclideanTransform, FundamentalMatrixTransform,
+                         PiecewiseAffineTransform, PolynomialTransform,
+                         ProjectiveTransform, SimilarityTransform,
+                         estimate_transform, matrix_transform)
+from ._thin_plate_splines import tps_transform, tps_warp
```

Comment:
Could you revert the unnecessary formatting changes here to keep the diff focused?